autogluon/utils/tabular/ml/models/lgb/hyperparameters/lgb_trial.py

```diff
@@ -17,40 +16,19 @@
 def lgb_trial(args, reporter):
     """ Training script for hyperparameter evaluation of Gradient Boosting model """
     try:
+        model, args, util_args = model_trial.prepare_inputs(args=args)
+
         try_import_lightgbm()
```

Comment:
should we consider importing lightgbm outside of the individual trial? Ie. in `hyperparameter_tune`

Reply:
Interesting idea, what would be gained by doing this?